src/main/java/org/opentest4j/ValueWrapper.java

```diff
@@ -49,12 +49,15 @@
 	 *
 	 * <p>If the supplied {@code value} is {@code null}, this method will return a
 	 * cached {@code ValueWrapper} suitable for all null values.
+	 * If the supplied {@code value} is already an instance of {@link ValueWrapper},
+	 * it will be returned as is.
 	 *
 	 * @param value the value to wrap; may be {@code null}
 	 * @return a wrapper for the supplied value
 	 */
 	public static ValueWrapper create(Object value) {
-		return (value == null ? nullValueWrapper : new ValueWrapper(value));
+		return (value instanceof ValueWrapper) ? (ValueWrapper) value
+				: ((value == null) ? nullValueWrapper : new ValueWrapper(value));
```

Comment:
I think an `if` statement for `value instanceof ValueWrapper` would be more readable in this case.